404.md

```diff
@@ -21,12 +19,11 @@ occurred, if applicable.
 let el = document.querySelector("a.newissue");
 if (el) {
     let url = new URL("{{ site.repo }}/issues/new");
-    url.searchParams.set("title", "404 at " + window.location.pathname);
-    let body = "I found a broken link : " + window.location.href;
-    if (document.referrer !== "") {
-        body += "\nI arrived on this page through: " + document.referrer
-    }
-    url.searchParams.set("body", body);
+    url.searchParams.set("template", "broken_link.yml")
+    url.searchParams.set("title", "[404]: " + window.location.pathname);
+    url.searchParams.set("target", window.location.href);
+    url.searchParams.set("location", document.referrer);
```

Comment:
oh, nice! didn't know you could pre-propagate individual fields as well

Reply:
Yeah it works real well for the broken link reports!

Reply:
We could even add a "needs more info" label if none is provided 😃

Reply:
I made location mandatory 😅